### DISCUSSION TITLE
Target Sensors not being prompted for on setup

### DISCUSSION THREAD

**Discussion by stav242** [BottlecapDave/HomeAssistant-OctopusEnergy#229] [General]
I've entered my API details on install, and can see my data, but the setup process does not prompt for the target sensors and none have been created. I do not see how to set them up.

Instructions say .
Target Rates
If you go through the setup process after you've configured your account, you can set up target rate sensors.

How should this be done please.

**Comment by BottlecapDave** [comment]
Hello sorry for the late response. I'm planning on updating the instructions, but you can add target rates by going into your integrations and then clicking "add integration" and searching for Octopus Energy again. When you add it this time (and future times) you'll be configuring target rate sensors. This is done this way due to a limitation on what can be achieved with the custom integration UIs in HA.
